### PR TITLE
Edit DartPad embeds, add troubleshooting instructions

### LINKED ISSUE #1891: Link to DartPad troubleshooting page
Every page that depends on DartPad should link to /tools/dartpad/troubleshoot. I'm thinking of searching for pages with embedded DartPads and including some text just before the topmost embedded DartPad.

Related: #1880, #1838

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #1891 
Contributes to #1825

Pages with embedded DartPads:
https://kw-www-dartlang-1.firebaseapp.com/guides/language/language-tour
https://kw-www-dartlang-1.firebaseapp.com/tutorials/language/streams#receiving-stream-events
https://kw-www-dartlang-1.firebaseapp.com/tutorials/server/get-started
https://kw-www-dartlang-1.firebaseapp.com/tutorials/web/fetch-data
https://kw-www-dartlang-1.firebaseapp.com/tutorials/web/get-started
https://kw-www-dartlang-1.firebaseapp.com/tutorials/web/low-level-html/add-elements#moving-elements
https://kw-www-dartlang-1.firebaseapp.com/tutorials/web/low-level-html/remove-elements
https://kw-www-dartlang-1.firebaseapp.com/codelabs/async-await
https://kw-www-dartlang-1.firebaseapp.com/codelabs/dart-cheatsheet

Other pages:
https://kw-www-dartlang-1.firebaseapp.com/tools/dartpad
https://kw-www-dartlang-1.firebaseapp.com/tools/dartpad/troubleshoot
https://kw-www-dartlang-1.firebaseapp.com/tutorials/web/low-level-html/connect-dart-html

### CODE REVIEW

**Comment by kwalrath** [comment]
I still need to add the troubleshooting instructions to other places.

**Comment by kwalrath** [comment]
Staged @ https://kw-www-dartlang-1.firebaseapp.com/. (I'll add all the page URLs to the first comment.)

File: _config.yml [hunk 1]
```diff
@@ -25,7 +25,9 @@ pub-api: https://pub.dev/documentation
 pub-pkg: https://pub.dev/packages
 lints: https://dart-lang.github.io/linter/lints
 dartpad: https://dartpad.dev
-dartpadx: https://dartpad.dev/experimental/embed-new-dart.html
+dartpad-embed: https://dartpad.dev/experimental/embed-new-dart.html
+dartpad-embed-html: https://dartpad.dev/experimental/embed-new-html.html
+dartpad-embed-inline: https://dartpad.dev/experimental/embed-new-inline.html
```
Comment (johnpryan) [inline _config.yml]: these should be:

- https://dartpad.dev/embed-dart.html
- https://dartpad.dev/embed-html.html
- https://dartpad.dev/embed-inline.html

the experimental/ directory will be removed at some point

Comment (kwalrath) [reply _config.yml]: ptal

**Pull request merged**